test/test.api.js

```diff
@@ -223,6 +223,68 @@ DRIVERS.forEach(function(driverName) {
             });
         });
 
+        if (driverName === localforage.INDEXEDDB) {
```

Comment:
Thanks for adding the `expect` calls, but I'm curious: what _exactly_ is this testing for? I'm a bit confused and the PR doesn't have a description nor is this related to an issue.

What bug does this prevent and what are these tests doing?